mixer/Makefile

```diff
@@ -1,4 +1,4 @@
-GO_MINOR_VERSION = $(shell go version | cut -d . -f 2)
+GO_VERSION=$(shell go version | sed "s/[a-z| ]*\([0-9]*\)\.\([0-9]*\).*/\1 \2/" | awk '{print $$1*100+$$2}')
```

Comment:
IMO magic incantations should have a short comment.  Not everyone is knowledgeable about regexp.